assets/js/components/VehicleTitle.vue

```diff
@@ -46,6 +54,29 @@ export default {
 			}
 			return this.$t("main.vehicle.none");
 		},
+		isUnknown() {
+			return this.connected && !this.vehiclePresent;
+		},
+		otherVehicles() {
+			return this.vehicles.filter((v) => v !== this.vehicleTitle);
```

Comment:
@naltatis This will return the wrong index for every vehicle after the currently selected. We need to maintain both index and title.